twoaxistracking/trackerfield.py

```diff
@@ -135,7 +135,7 @@ def get_shaded_fraction(self, solar_elevation,  solar_azimuth,
                             plot=False):
         """Calculate the shaded fraction for the specified solar positions.
 
-        Uses the :py:func:`twoaxistracking.shaded_fraction` function to
+        Uses the :py:func:`tracking.shaded_fraction` function to
```

Comment:
Right. I have changed it to:

    Uses the :py:func:`shading.shaded_fraction` function to

though I guess it could also be `twoaxistracking.shaded_fraction` because it is available in both places. Any preference?

Reply:
I'm not sure the link will work unless you call it the same thing as the `autosummary::` calls it.  So `twoaxistracking.shaded_fraction` I think is better.  We should check that it links correctly in the build docs.